src/Service/Sns/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Allow passing explicit null values for optional fields of input objects
 
+### Added
```

Comment:
Isn't this a BC break?

Reply:
If people calls `throw ValidationException` yes it could be.
But if they only use `catch (ValidationException $e)` it will be OK, because the class doesn't need to exist.

I believe the first case is hypothetical and people don't do that.. :thinking: